### ISSUE TITLE
[Feature Request] Load URL in iframe as widget

### ISSUE THREAD

**Issue by tnyeanderson** [nextcloud/dashboard#53]
There should be an option in the "Add New Widget" popup to load any URL (similar to how "external sites" works). If the page is on the same domain (i.e. is another Nextcloud page) allow the running of custom scripts as well. This could be added as a Widget to the nextcoud/external app, or it could be added here. I think adding it here would be a much smoother experience.

One problem this could fix rather quickly is that there aren't many supported apps. If I could just load the nextcloud URL to my calendar, then style it to remove the menubar and sidebar, I can get it to work without waiting for nextcloud/calendar to add the widget.

**Comment by MarkPartlett** [comment]
Like I have done here? https://youtu.be/3TzOXOcAV2I

**Comment by tnyeanderson** [comment]
Yes! And I love the music. How did you get that working? I watched the video but don't see the same options. Maybe I'm just missing something?

**Comment by MarkPartlett** [comment]
Still WIP https://github.com/MarkPartlett/dashboardcharts

**Comment by tnyeanderson** [comment]
Looks great! Though I wonder: why have a seperate app? Shouldn't the changes be implemented here? At the very least, iFrame functionality built in would be nice, and the graphs could stay as a separate app. 

Thanks for the hard work!
